src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -70,7 +70,7 @@ public function register() {
 
 		register_rest_route(
 			$namespace,
-			$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)',
+			$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)(?:/(?P<payer_id>[0-9a-zA-Z]+))?',
```

Comment:
Why are we using this:
```$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)(?:/(?P<payer_id>[0-9a-zA-Z]+))?'```

Instead of this:
```$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)/(?P<payer_id>[0-9a-zA-Z]+)'```

??

Reply:
That makes the endpoint `payer_id` optional. So, both `/order/order_id` and `order/order_id/payer_id` will be handled by the same route.

But we don't need this anymore now, as we are using body params.